pipeline/00-ingest.R

```diff
@@ -156,6 +158,29 @@ recode_column_type <- function(col, col_name, dict = col_type_dict) {
 }
 
 
+# Mini function to deal with arrays
+# Some Athena columns are stored as arrays but are converted to string on
+# ingest. In such cases, we either keep the contents of the cell (if 1 unit),
+# collapse the array into a comma-separated string (if more than 1 unit),
+# or replace with NA if the array is empty
+
+process_array_columns <- function(data) {
+  data %>%
+    mutate(
+      across(
+        starts_with("loc_tax_"),
```

Comment:
**suggestion (blocking):** It's generally poor form to have implicit stuff like this inside of functions. I would modify this function to take an argument i.e. `selector` and use that in place of `starts_with("loc_tax_")`, then pass the `starts_with` as an argument.